modules/qtwidgets/src/properties/propertywidgetqt.cpp

```diff
@@ -64,6 +64,13 @@ const int PropertyWidgetQt::minimumWidth = 250;
 const int PropertyWidgetQt::spacing = 7;
```

Comment:
what is the reason for a `minimumWidth` of 250? Can't we reduce this to maybe 220 or even 200?

Reply:
(defined in line 63)

Reply:
Think a lot of outpr widgets will look very cramped then, but we can test...

Reply:
I've put in 200. There is no difference since you can squeeze the property list only down to 330px. (Win 10, Qt 5.12, no dpi scaling).